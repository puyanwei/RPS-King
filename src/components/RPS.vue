<template>
  <h1>RPS King</h1>
</template>

<script>
export default {};
</script>

<style>
</style>
