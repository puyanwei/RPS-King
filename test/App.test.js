import { shallow } from "vue-test-utils";
import RPS from "../src/components/RPS";

describe("RPS component", () => {
  let cmp;

  beforeEach(() => {
    cmp = shallow(RPS, {});
  });

  it("has the expected html structure", () => {
    expect(cmp.element).toMatchSnapshot();
  });
});
