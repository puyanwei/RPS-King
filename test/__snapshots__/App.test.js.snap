// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RPS component has the expected html structure 1`] = `
<h1>
  RPS King
</h1>
`;
